core/src/util/usProperties_p.h

```diff
@@ -0,0 +1,57 @@
+/*=============================================================================
+
+  Library: CppMicroServices
+
+  Copyright (c) The CppMicroServices developers. See the COPYRIGHT
+  file at the top-level directory of this distribution and at
+  https://github.com/saschazelzer/CppMicroServices/COPYRIGHT .
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+=============================================================================*/
+
+#ifndef USPROPERTIES_P_H
+#define USPROPERTIES_P_H
+
+#include "usServiceProperties.h"
```

Comment:
headers should be self sufficient (i.e. they include everything they need for their declaration). See #66 for reasons why.

You should add ```<vector>```, ```<map>```, ```<string>``` and ```Any``` to satisfy the declaration.